pycbc/io/live.py

```diff
@@ -202,7 +203,7 @@ def __init__(self, coinc_ifos, ifos, coinc_results, **kwargs):
         coinc_inspiral_row.end_time = sngl_populated.end_time
         coinc_inspiral_row.end_time_ns = sngl_populated.end_time_ns
         coinc_inspiral_row.snr = network_snrsq ** 0.5
-        far = 1.0 / (lal.YRJUL_SI * coinc_results['foreground/ifar'])
+        far = conv.sec_to_year(1.0 / coinc_results['foreground/ifar'])
```

Comment:
we are usually writing '1.' for some reason ..
```suggestion
        far = conv.sec_to_year(1. / coinc_results['foreground/ifar'])
```